lib/checker.js

```diff
@@ -1,6 +1,7 @@
 var vowFs = require('vow-fs');
 var Vow = require('vow');
 var StringChecker = require('./string-checker');
+var extractJs = require('./extract-js');
```

Comment:
So this class called as `HtmlFile` analogues to `JsFile`, but file named as `extract-js`?